analyses/tmb-compare-tcga/util/tmb-plot-function.R

```diff
@@ -0,0 +1,62 @@
+# Function for TMB CDF plot
+#
+# C. Savonen for ALSF - CCDL
+
+tmb_cdf_plot <- function(tmb_df,
+                         plot_title,
+                         color) {
+  # Print CDF TMB plot
+  #
+  # Args:
+  #   tmb_df: a data.frame with `tmb` and `short_histology` as column names.
+  #   plot_title: A string to be passed to "ggtitle".
+  #   color: What color should the points be. Will be passed to `geom_point`. 
+  #
+  # Returns:
+  # A CDF plot of TMB with histology
+  tmb_df %>%
+    as.data.frame() %>%
+    dplyr::mutate(short_histology = tools::toTitleCase(short_histology)) %>%
+    # Only plot histologies groups with more than `min_samples` number of samples
+    dplyr::group_by(short_histology, add = TRUE) %>%
+    # Only keep groups with this amount of samples
+    dplyr::filter(dplyr::n() > 5) %>%
```

Comment:
This should be an argument for this function rather than hardcoded and not included in the documentation.